target/linux/imx6/lumi/base-files/etc/uci-defaults/50_disable-removing-wdev

```diff
@@ -0,0 +1,5 @@
+. /lib/functions.sh
+
+sed -i 's/iw dev "$wdev" del/# iw dev "$wdev" del/' /lib/netifd/wireless/mac80211.sh
```

Comment:
This should be fixed properly somehow instead.

Reply:
Removed this patch from the PR